### PR TITLE
adding bootstrap and consistent nav

### LINKED ISSUE #34: Make a consistent navbar
https://www.figma.com/proto/vYNmE2bvxZbpwjFbv2dS2I/BNBIntel?page-id=190%3A5694&type=design&node-id=190-7324&viewport=-103%2C705%2C0.3&t=aNh7DfPQ10bbncjt-1&scaling=min-zoom&starting-point-node-id=190%3A7324

Let's make a navbar that is consistent across all of the bnbintel pages (this will probably require rewriting a lot of the blazer views, but that's ok).

Let's put the options that you see in the navbar in the linked Figma. Let's also add a configuration setting for setting a logo so we can pull in the application logo

### PR DESCRIPTION
**Pull request by ianfletcher314**
closes #34

UX:

https://github.com/brandnewbox/bnbintel/assets/77754482/5ae37fba-9c94-48f2-ab78-a85db4949892

### CODE REVIEW

File: app/views/blazer/_new_nav.html.erb [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    document.getElementById('secondary-sidebar').classList.remove('d-none');
+    document.getElementById('secondary-sidebar').classList.add('d-block');
+  });
+  document.getElementById('secondary-close-btn').addEventListener('click', function() {
+    document.getElementById('sidebar').classList.remove('align-start');
+    document.getElementById('primary-sidebar').classList.remove('d-none');
+    document.getElementById('primary-sidebar').classList.add('d-block');
+    document.getElementById('secondary-sidebar').classList.remove('d-block');
+    document.getElementById('secondary-sidebar').classList.add('d-none');
+  });
```
Comment (willtcarey) [inline app/views/blazer/_new_nav.html.erb]: Since we have a build now, let's do this as a stimulus controller

File: app/views/blazer/_new_nav.html.erb
Comment (willtcarey) [inline app/views/blazer/_new_nav.html.erb:1]: Let's put this inside of bnb_blazer folder and just call it `_nav` and update any references necessary

**Review by willtcarey — CHANGES_REQUESTED** [review]

File: Gemfile.lock
Comment (willtcarey) [inline Gemfile.lock:1]: We should undo the changes on this file.We didn't wind up keeping this file

File: app/views/blazer/queries/show.html.erb [hunk 2]
```diff
@@ -37,7 +26,20 @@
...
+    </div>
+    <div class="col d-flex justify-content-end">
+      <%= link_to "Edit", edit_query_path(@query, variable_params(@query)), class: "btn bg-primary rounded-pill mx-2 text-white", disabled: !@query.editable?(blazer_user) %>
+      <%= link_to "Fork", new_query_path(variable_params(@query).merge(fork_query_id: @query.id, data_source: @query.data_source, name: @query.name)), class: "btn bg-primary rounded-pill mx-2 text-white" %>
+      <% if !@error && @success %>
+        <%= button_to "Download", run_queries_path(format: "csv"), params: run_data, class: "btn bg-primary rounded-pill mx-2 text-white" %>
+      <% end %>
+    </div>
+  </div>
+</div>
```
Comment (willtcarey) [inline app/views/blazer/queries/show.html.erb]: This all seems like it should go above the `margin-bottom` up above?

Comment (ianfletcher314) [reply app/views/blazer/queries/show.html.erb]: I moved the content above the margin-bottom but when I do that the new card hugs the nav:
image

when they content gos bellow the margin bottom the card appears as part of the main content which @vikascripps had suggested:
image

Comment (willtcarey) [reply app/views/blazer/queries/show.html.erb]: Ok, that seems fine to have it go below the bottom margin, but I still think it should be moved above the error and description sections

**Review by willtcarey — COMMENTED** [review]
See above

**Review by willtcarey — APPROVED** [review]

**Pull request merged**